src/components/Sections.js

```diff
@@ -15,4 +15,8 @@ export default class Section {
   addItem(element) {
     this._section.prepend(element);
   }
+
+  setItems(item) {
```

Comment:
1) There is no need to have this method
2) If this method is called setItems then it should set items, not a single item